README.rst

```diff
@@ -13,7 +13,13 @@ __ https://docs.us-phoenix-1.oraclecloud.com/Content/API/Concepts/cliconcepts.ht
 Installation
 ============
 
-Mac / Linux
+Oracle Linux 7.x
```

Comment:
I think this is true for all Oracle Linux versions. We do not need a version number here

Reply:
FYI, we run tests for Oracle Linux 6.8 and 6.9 as well.